main.py

```diff
@@ -0,0 +1,103 @@
+""" SHEST FASTAPI SERVER """
+
+from cryptography.fernet import Fernet
+from fastapi import FastAPI, Request
+from os import popen
+from uvicorn import run
+from fastapi.responses import StreamingResponse
+
+
+app = FastAPI()
+
+
+def load_key():
+    """Load the previously generated key"""
+
+    with open('secret.key', 'r') as file:
+        return file.read()
+
+
+def encrypt_message(message: str):
+    """
+    Encrypts a message"
+    :param message: A regular message.
+    :return: The encrypted message.
+    """
+
+    key = load_key()
+    f = Fernet(key)
+    message = message.encode()
+    encrypted_message = f.encrypt(message)
+    return encrypted_message
+
+
+def decrypt_message(encrypted_message):
+    """
+    Decrypts an encrypted message
+    :param encrypted_message: The encrypted message to decrypt.
+    :return: The decrypted message.
+    """
+
+    key = load_key()
+    f = Fernet(key)
+    decrypted_message = f.decrypt(encrypted_message)
+    return decrypted_message
+
+
+async def stream_back(file_name: str):
+    """
+    Yield file content in chunks
+    :param file_name: A file name.
+    :return: Chunks of a file.
+    """
+
+    with open(f"C:\\Users\\Elad\\PycharmProjects\\SHEST\\"
+              f"files\\{file_name}", 'r') as file:
```

Comment:
I don't have this path in my PC. Use relative paths to the SHEST application. Export the path to a constant too.

Reply:
Apply in `upload` too.